managed/SavedSearchFundingCases.mgd.php

```diff
@@ -83,7 +84,7 @@
   [
     'name' => 'SavedSearch_funding_cases_SearchDisplay_table',
     'entity' => 'SearchDisplay',
-    'cleanup' => 'unused',
+    'cleanup' => 'always',
```

Comment:
not clear why this field was set to always. this could have happend during export?

Reply:
switched back to _unused_